Dockerfile

```diff
@@ -135,7 +133,7 @@ COPY ./macros.d /usr/lib/rpm/macros.d
 ARG JENKINS_USERNAME=jenkins
 ENV USER=${JENKINS_USERNAME}
 ENV HOME=/home/"${JENKINS_USERNAME}"
-RUN useradd -m -u 1000 "${JENKINS_USERNAME}"
+RUN deluser ubuntu && useradd -m -u 1000 "${JENKINS_USERNAME}"
```

Comment:
The new Ubuntu image has an `ubuntu` user with UID 1000, so we have to delete this user first before we can use that UID.